ts/features/walletV3/onboarding/components/WalletOnboardingWebView.tsx

```diff
@@ -23,7 +23,7 @@ import { extractOnboardingResult } from "../utils";
 type WalletOnboardingWebViewProps = {
   paymentMethodId: string;
   onSuccess: (outcome: OnboardingOutcome, walletId: string) => void;
```

Comment:
I think this should have a `OnboardingOutcomeSuccess` as `outcome` parameter. Am I right?
```suggestion
  onSuccess: (outcome: OnboardingOutcomeSuccess, walletId: string) => void;
```

Reply:
Addressed into c883a57e67f7400e8946946e9422786ffd83fd7c